utility/src/main/java/org/oppia/android/util/parser/image/BUILD.bazel

```diff
@@ -0,0 +1,225 @@
+    ],
+)
+
+kt_android_library(
+    name = "glide_image_loader",
+    srcs = [
+        "GlideImageLoader.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+    deps = [
+        ":image_asset_fetcher",
+        ":image_loader",
+        ":image_targets",
+        "//third_party:com_github_bumptech_glide_glide",
+        "//third_party:javax_inject_javax_inject",
+        "//utility/src/main/java/org/oppia/android/util/caching:annotations",
+        "//utility/src/main/java/org/oppia/android/util/caching:asset_repository",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:block_picture_drawable",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:scalable_vector_graphic",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:svg_blur_transformation",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:svg_decoder",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:svg_picture_drawable",
+    ],
+)
+
+kt_android_library(
+    name = "glide_image_loader_module",
+    srcs = [
+        "GlideImageLoaderModule.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+    deps = [
+        ":dagger",
+        ":glide_image_loader",
+        ":image_loader",
+    ],
+)
+
+kt_android_library(
+    name = "image_asset_fetcher",
+    srcs = [
+        "ImageAssetFetcher.kt",
+    ],
+)
+
+kt_android_library(
+    name = "test_glide_image_loader",
+    srcs = [
+        "TestGlideImageLoader.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+    deps = [
+        ":glide_image_loader",
+        ":image_loader",
+        ":text_picture_drawable",
+        "//third_party:javax_inject_javax_inject",
+        "//utility/src/main/java/org/oppia/android/util/parser/svg:block_picture_drawable",
+    ],
+)
+
+kt_android_library(
+    name = "image_parsing_annonations",
+    srcs = [
+        "ImageParsingAnnotations.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+    deps = [
+        "//third_party:javax_inject_javax_inject",
+    ],
+)
+
+kt_android_library(
+    name = "image_parsing_module",
+    srcs = [
+        "ImageParsingModule.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+    deps = [
+        ":dagger",
+        ":image_parsing_annonations",
+        "//third_party:javax_inject_javax_inject",
+    ],
+)
+
+kt_android_library(
+    name = "image_targets",
+    srcs = [
+        "ImageTargets.kt",
+    ],
+    visibility = ["//:oppia_testing_visibility"],
+    deps = [
+        ":image_asset_fetcher",
+        "//third_party:com_github_bumptech_glide_glide",
+    ],
+)
+
+kt_android_library(
+    name = "image_transformation",
+    srcs = [
+        "ImageTransformation.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+)
+
+kt_android_library(
+    name = "repository_glide_module",
```

Comment:
``//app`` also needs to depend on this or image loading won't work correctly in certain cases (specifically SVGs).

Reply:
this is being used in only three places which are auto-generated dagger code and therefore I didn't added it.
![image](https://user-images.githubusercontent.com/76404817/145676061-5c122da5-64a3-4e1b-a913-eb0e957065eb.png)

Reply:
so i am not adding any dependencies in this for this module

Reply:
To clarify, that's actually the thing that you need to do. We need to add repository_glide_module as a dependency somewhere in the build graph (I recommend "//app" to keep it simple).